### ISSUE TITLE
Why is get_velocity applied to model_output during the training of cogvideo?

### ISSUE THREAD

**Issue by JH-Xie** [THUDM/CogVideo#403]
### Feature request  / 功能建议

The common way of writing is:
：
```
if noise_scheduler.config.prediction_type == "epsilon":
    target = noise
elif noise_scheduler.config.prediction_type == "v_prediction":
    target = noise_scheduler.get_velocity(latents, noise, timesteps)
```
This applies get_velocity to target.
However, the training code for cogvideo applies get_velocity to model_output like this:
`model_pred = scheduler.get_velocity(model_output, noisy_video_latents, timesteps)`
Why is this different from the approach described in the paper? And why does it work?

### Motivation / 动机

/

### Your contribution / 您的贡献

/
